Makefile.pseudomodules

```diff
@@ -2,3 +2,4 @@ PSEUDOMODULES += defaulttransceiver
 PSEUDOMODULES += transport_layer
 PSEUDOMODULES += pktqueue
 PSEUDOMODULES += ng_netbase
+PSEUDOMODULES += newlib
```

Comment:
Don't think so.

I did this while debugging why this commit enlarges the binary (turned out to be the RIOT_VERSION string), but then I thought, it is not needed there, so why add it there.

You guys opt for not making this a pseudomodule?